README.md

```diff
@@ -112,6 +112,29 @@ BITBUCKET_HOSTNAME     | `bitbucket.org`
 
 \* Either `BITBUCKET_ACCESS_TOKEN` must be passed, or `BITBUCKET_APP_USERNAME` and `BITBUCKET_APP_PASSWORD`.
 
+**Private repository credentials**
+
+Variable               | Default
+:------                | :------
+PRIVATE_REPOSITORIES   | N/A (Optional)
```

Comment:
`PRIVATE_REPOSITORIES` feels like a confusing name here - isn't this feature really about **registry** credentials? Perhaps `PRIVATE_REGISTRY_CREDENTIALS` would be clearer?